### PR TITLE
Misc/add yaml linter

### PR DESCRIPTION
**Pull request by sabrinaheid**
## Preflight Checklist

* [ ] I have read the [Contributing Guidelines](https://github.com/lyne-design-system/lyne-components/blob/master/.github/CONTRIBUTING.md) for this project.
* [ ] I agree to follow the [Code of Conduct](https://github.com/lyne-design-system/lyne-components/blob/master/.github/CODE_OF_CONDUCT.md) that this project adheres to.
* [ ] I have searched the [pull request tracker](https://github.com/lyne-design-system/lyne-components/pulls) for a Pull Request (PR) that matches the one I want to submit, without success.

## Issue

This PR Closes #

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [ ] Build (`npm run build`) was run locally and any changes were pushed
- [ ] Lint (`npm run lint`) has passed locally and any fixes were made for failures

## Changes

Changes in this pull request:

-

## Browsers

I tested the build on the following browsers:
- [ ] Firefox Desktop
- [ ] Chrome Desktop
- [ ] Safari Desktop
- [ ] Firefox Mobile
- [ ] Chrome Mobile
- [ ] Safari Mobile
- [ ] Opera
- [ ] Brave

## Pull request type

Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## Does this introduce a breaking change?

- [ ] Yes
- [ ] No

## Mentions

@lyne-design-system/core

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -1,3 +1,7 @@
 /src/components.d.ts
 dist
-storybook-static
+storybook
```
Comment (feerglas) [inline .eslintignore:3]: `storybook-static` war schon richtig

File: package.json [hunk 2]
```diff
@@ -28,6 +28,7 @@
     "chromatic:master": "chromatic --storybook-build-dir=./storybook-static --exit-zero-on-changes --auto-accept-changes",
     "generate": "stencil generate",
     "lint": "eslint ./ --ext .ts,.tsx",
+    "lint:yml": "eslint ./ --ext .yml",
```
Comment (feerglas) [inline package.json:31]: Vielleicht wäre es doch gut, wenn wir den yml-linter immer laufen lassen, sonst geht uns der wahrscheinlich durch die lappen ;)
"lint": "npm run lint:js && npm run lint:yml",
"lint:js": "eslint ./ --ext .ts,.tsx",
"lint:yml": "eslint ./ --ext .yml",

File: .eslintignore (hunk 1 above)
Comment (sabrinaheid) [reply .eslintignore:3]: das ist ein Versehen. Danke

Comment (sabrinaheid) [reply .eslintignore]: `npm install` habe ich nochmals gemacht und das aktuellste package-lock hinzugefügt:-)

**Pull request merged**